packages/chat-headless/src/slices/conversation.ts

```diff
@@ -2,7 +2,11 @@ import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { ConversationState } from "../models/slices/conversation";
 import { Message, MessageNotes } from "@yext/chat-core";
 
-export const initialState: ConversationState = {
+export const STATE_SESSION_STORAGE_KEY : string = "conversationState";
+export const initialState: ConversationState =
+    sessionStorage.getItem(STATE_SESSION_STORAGE_KEY)
```

Comment:
sessionStorage may be undefined if user/browser defined access permission or if it's server side rendering